src/data/testData.js

```diff
@@ -114,7 +114,21 @@ const testIngredients = [
     id: 20081,
     name: "wheat flour",
     estimatedCostInCents: 142,
-    amount: 1.5,
+    amount: 5,
+    unit: "cups"
+  },
+  {
+    id: 20081,
+    name: "wheat flour",
+    estimatedCostInCents: 142,
+    amount: 5,
+    unit: "cups"
+  },
+  {
+    id: 20081,
+    name: "wheat flour",
+    estimatedCostInCents: 142,
+    amount: 5,
```

Comment:
That's actually leftover from my last PR where I was testing dupe ingredients. I duplicated wheat flour in the test data to see if the function would remove it. I didn't think it would cause any issues but I can certainly remove the dupes. Let me know what you think is best.

Reply:
Nope that's great! Just wanted to make sure it hadn't sneaked in by accident.